src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -46,6 +47,16 @@ public string CookieName
         }
 
         /// <summary>
+        /// The path set on the cookie. Defaults to current request's <see cref="HttpRequest.PathBase"/> value.
```

Comment:
I would say that - if it's null, we use the current request's pathbase. It's not that **this** property will be initialized to the pathbase